chromium_src/chrome/common/url_constants.cc

```diff
@@ -23,6 +23,8 @@ const char kAutomaticSettingsResetLearnMoreURL[] =
 const char kAdvancedProtectionDownloadLearnMoreURL[] =
     "https://support.brave.com";
 
+const char kBatterySaverModeLearnMoreUrl[] = "https://support.brave.com";
```

Comment:
i would append trailing `/` here and above just for consistency :D

Reply:
Done via 420ff1e07c